workshops/drumpad/README.md

```diff
@@ -0,0 +1,454 @@
+---
+name: 'Drum Pad'
+description: 'Creating a Drum Pad in HTML, CSS & JS'
+author: '@emmanuel39hanks'
+---
+
+# Creating a Drum Pad in HTML, CSS & JavaScript
+
+In this workshop, we are going to be creating a drum pad in less than 230 lines of code.
+
+_Preview of the Drum Pad we are going to be creating_
+
+![Drum Pad Preview](https://cloud-ms6ubs6qu.vercel.app/0screencapture-drum-emmanuel39hanks-repl-co-2020-11-07-23_20_37.png)
+
+[Link to demo](https://drum.emmanuel39hanks.repl.co)
+
+## Getting started
+
+Head over to repl.it (https://repl.it/) and we will get started by creating an HTML/CSS & JavaScript repl. Once the repl is set up, navigate to your `index.html` file, and we can start working on the pad layout. 
+
+## Part 1
+
+We are going to start writing HTML; going into our `<body>` tag, which is where we write code that will be shown to the user of the web page. Let's create a header with the `<h1>` tag. Then we will have three rows and four columns of pads and each pad will be a `<div>`. you can think of the `<div>` tag as a box or container.
+
+We will do that with the following code:
+
+```html
```

Comment:
Maybe consider breaking up the code into smaller chunks? For example, go over the code for the parent div and move the comments into a paragraph outside of the code, rather than including them in the code where they're easy to miss?